app/components/ui/SanitizedUserContent.tsx

```
@@ -34,7 +34,10 @@ const SanitizedUserContent: React.FC<SanitizedUserContentProps> = ({
   return (
     <div
       className={className}
-      dangerouslySetInnerHTML={{ __html: sanitizedHtml }}
+      dangerouslySetInnerHTML={useMemo(
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)